app/src/main/res/values/strings.xml

```diff
@@ -1287,6 +1287,11 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_postboxRoyalCypher_title">What’s the royal cypher on this postbox?</string>
     <string name="quest_postboxRoyalCypher_type_none">No royal cypher is visible</string>
     <string name="quest_postboxRoyalCypher_type_scottish_crown">The Crown of Scotland</string>
+	
+	<string name="quest_powerAttachment_title">How power line is attached on this support?</string>
+	<string name="quest_powerAttachment_suspension">Suspension</string>
+	<string name="quest_powerAttachment_anchor">Anchor</string>
+	<string name="quest_powerAttachment_pin">Pin</string>
```

Comment:
Please use spaces for indentation, not tabs. Also, I suggest this wording for the question:

```suggestion
    <string name="quest_powerAttachment_title">How is this power line attached?</string>
    <string name="quest_powerAttachment_suspension">Suspension</string>
    <string name="quest_powerAttachment_anchor">Anchor</string>
    <string name="quest_powerAttachment_pin">Pin</string>
```